dockstore-webservice/src/main/java/io/dockstore/webservice/resources/UserResource.java

```diff
@@ -718,6 +727,50 @@ private void checkToolTokens(User authUser, Long userId, String organization) {
         return userDAO.findAll();
     }
 
+    // TODO: Do equivalent of below, but for Google users.
+    @GET
+    @Timed
+    @UnitOfWork
+    @RolesAllowed("admin")
+    @Path("/updateUserMetadataToGetIds")
+    @Deprecated
+    @Operation(operationId = "updateUserMetadataToGetIds", description = "Update metadata of all GitHub users and returns list of Dockstore users who could not be updated.", security = @SecurityRequirement(name = OPENAPI_JWT_SECURITY_DEFINITION_NAME))
+    @ApiResponse(responseCode = HttpStatus.SC_OK + "", description = "Get list Dockstore users we were unable to get GitHub IDs for.", content = @Content(
+            mediaType = "application/json",
+            array = @ArraySchema(schema = @Schema(implementation = User.class))))
+    @ApiOperation(value = "See OpenApi for details", hidden = true)
+    public List<User> updateUserMetadataToGetIds(@ApiParam(hidden = true) @Parameter(hidden = true, name = "user")@Auth User user) {
+        List<Token> gitHubTokens = tokenDAO.findAllGitHubTokens();
+        List<User> usersNotUpdatedWithToken = new ArrayList<>();
+        List<User> usersNotUpdatedWithTokenOrUsername = new ArrayList<>();
+
+        for (Token t : gitHubTokens) {
+            User currentUser = userDAO.findById(t.getUserId());
+            if (currentUser != null) {
```

Comment:
Is it possible for the user to be null? I don't think so, because there's a foreign key with token.userId and endUser. It's fine to leave the test, just trying to make sure I understand.

Reply:
I don't think it is possible